### PR TITLE
Adding guide to create personal vpn tunnel via OpenVPN Access Server with Ubuntu.

### PR DESCRIPTION
**Pull request by yoneal**
Easily setup your own VPN tunnel server with OpenVPN Access Server. The guide uses Ubuntu 14.04 (Trusty) and UFW.

### CODE REVIEW

**Comment by seanthewebber** [comment]
FYI... this *may* be a duplicate of #306. Pritunl is basically another GUI for OpenVPN, but @cwlinode will have to decide.

**Comment by alexfornuto** [comment]
Does this guide include any information not in [this guide](https://www.linode.com/docs/networking/vpn/openvpn-access-server)?

**Comment by seanthewebber** [comment]
@alexfornuto Comparing that tutorial... nope, this is definitely a duplicate. Sorry @yoneal :/

**Comment by alexfornuto** [comment]
Thanks for the input @seanthewebber but I'm actually looking for the author's feedback on that question.

**Comment by yoneal** [comment]
@alexfornuto It augments the [guide](https://www.linode.com/docs/networking/vpn/openvpn-access-server) to use the vpn tunnel for accessing the Internet; useful if you want to evade censorship, geolocation or company firewall.

**Comment by brewn** [comment]
@yoneal Taking a look at this now - just to clarify, is this a guide to tunnel _only_ web traffic, not other connections such as SSH, telnet, etc?

Thanks!

**Comment by alexfornuto** [comment]
@yoneal Just a reminder, you have a PR from Brewn waiting for you to review / merge.

**Comment by yoneal** [comment]
@brewn This also allows other traffic, not just web traffic.

**Comment by alexfornuto** [comment]
@pbzona has made a PR for you here: https://github.com/yoneal/docs/pull/2

**Comment by alexfornuto** [comment]
In addition to my single comment review, please review and merge https://github.com/yoneal/docs/pull/3

File: docs/networking/vpn/tunnel-internet-traffic-with-openvpn-access-server.md [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+
+## Configure OpenVPN Access Server for Tunneling
+
+To configure OpenVPN for tunneling, you'll first need to log in to the Access Server Admin UI, and navigate to the **VPN Settings** page.
+
+1. In the **Routing** section, ensure that the option "Should client Internet traffic be routed through the VPN?" is set to **Yes**.
+
+    ![OpenVPN Access Server Internet Routing.](/docs/assets/openvpn-access-server-routing.png)
+
+    The option "Should VPN clients have access to private subnets (non-public networks on the server side)?" can be set to **No**, since we are using the VPN to mask internet traffic. If you wish to give VPN users access to services listening on your Linode's local network, set this option to **Yes**.
```
Comment (yoneal) [inline docs/networking/vpn/tunnel-internet-traffic-with-openvpn-access-server.md]: I was referring to the same option "Should VPN clients have access to private subnets..".

Comment (alexfornuto) [inline docs/networking/vpn/tunnel-internet-traffic-with-openvpn-access-server.md]: What I mean is; if they want to give users access to services listening on Linode's local network, should they select **Yes, using NAT** or **Yes, using routing (advanced)**, or does that depend on additional configurations outside the scope of this guide?

**Comment by alexfornuto** [comment]
Hi @yoneal, have you had a chance to review my edit, or any of the most recent comments on this PR?

**Comment by yoneal** [comment]
Hi @alexfornuto , I have merged the PR.

**Comment by alexfornuto** [comment]
Thanks @yoneal. I'm still looking for an update on L38.

File: docs/networking/vpn/tunnel-internet-traffic-with-openvpn-access-server.md (hunk 1 above)
Comment (alexfornuto) [inline docs/networking/vpn/tunnel-internet-traffic-with-openvpn-access-server.md]: @yoneal any update?

Comment (yoneal) [inline docs/networking/vpn/tunnel-internet-traffic-with-openvpn-access-server.md]: Hi @alexfornuto, sorry I was swamped at work.
I think it is already outside the scope of this guide. But if they want to access serices on LInode's local network, the simplest would be to select **Yes, using NAT**.

**Comment by alexfornuto** [comment]
Thanks for the update! Before we publish this guide, I'd like to know how you'd prefer your bounty payment, as Linode Credit or through PayPal. In either case, you can let me know here or email the details to contribute@linode.com.

**Comment by alexfornuto** [comment]
Hey @yoneal, we're all ready to publish this guide! Be sure to send along those payment details so we can give you your bounty.

**Comment by yoneal** [comment]
Hi @alexfornuto , I sent an email to contribute@linode.com with the PayPal account for the bounty payment. Thanks in advance!

**Comment by alexfornuto** [comment]
Heads up, this *should* have been shown as merged when #587 was merged. I think you can safely close it now.

**Comment by cwlinode** [comment]
Thanks, Alex. Payment was sent on Nov. 4, 2016. Closing this.

**Comment by alexfornuto** [comment]
bitmoji

**Pull request closed without merging**